src/PollingBlockTracker.ts

```diff
@@ -1,38 +1,39 @@
+import type { SafeEventEmitterProvider } from '@metamask/eth-json-rpc-provider';
+import type { JsonRpcRequest } from 'json-rpc-engine';
 import getCreateRandomId from 'json-rpc-random-id';
 import pify from 'pify';
-import { JsonRpcRequest } from 'json-rpc-engine';
-import type { SafeEventEmitterProvider } from '@metamask/eth-json-rpc-provider';
+
 import { BaseBlockTracker } from './BaseBlockTracker';
 import { projectLogger, createModuleLogger } from './logging-utils';
 
 const log = createModuleLogger(projectLogger, 'polling-block-tracker');
 const createRandomId = getCreateRandomId();
 const sec = 1000;
 
-export interface PollingBlockTrackerOptions {
+export type PollingBlockTrackerOptions = {
   provider?: SafeEventEmitterProvider;
   pollingInterval?: number;
   retryTimeout?: number;
   keepEventLoopActive?: boolean;
   setSkipCacheFlag?: boolean;
   blockResetDuration?: number;
   usePastBlocks?: boolean;
-}
+};
 
-interface ExtendedJsonRpcRequest<T> extends JsonRpcRequest<T> {
+type ExtendedJsonRpcRequest<T> = {
   skipCache?: boolean;
-}
+} & JsonRpcRequest<T>;
```

Comment:
Another instance of the ESLint rule being broken here — what do you think about:

``` suggestion
type ExtendedJsonRpcRequest<T> = JsonRpcRequest<T> & {
  skipCache?: boolean;
};
```

Reply:
Okay, good call. Maybe worth revisiting that rule...

Reply:
Hmm, I don't see those reverts. Should this have been reverted?